lighthouse-core/runner.js

```diff
@@ -52,10 +52,13 @@ class Runner {
       // that have been listed in the config.
       const filteredPasses = config.passes.map(pass => {
         pass.gatherers = pass.gatherers.filter(gatherer => {
+          if (typeof gatherer !== 'string') {
```

Comment:
Here, I think it is used to check if the gatherer exists. If it doesn't then quickly throw, or filter out those things that are not required. The `require`d gatherer is not used here at all other than to verify it exists.